paddle/fluid/operators/jit/test.cc

```diff
@@ -761,7 +821,15 @@ TEST(JITKernel, kNCHW16CMulNC) {
   TestNCHW16CMulNCKernel<jit::kNCHW16CMulNC, double, CPUPlace>();
 }
 
-// TODO(yihua/TJ): add crf decoding and layer norm unit tests
+TEST(JITKernel, kLayerNorm) {
+  namespace jit = paddle::operators::jit;
```

Comment:
This line can be removed, since Line 63.

Same as Line 830.

Reply:
Done